.husky/pre-commit

```diff
@@ -2,4 +2,5 @@
 echo "pre-commit"
 . "$(dirname "$0")/_/husky.sh"
 
-npx lint-staged --allow-empty
\ No newline at end of file
+npx lint-staged --allow-emptyyarn tsc --noEmit && yarn eslint . && yarn prettier --write .
+yarn lint-staged .
```

Comment:
Maybe we can add `yarn build` here to check code styles for production mode.

Reply:
Sound good @sunshine1004-dev